Failing CI because of ignored return value of Utilities::MPI::broadcast().

The CI checks for #6383 fails because of this:
```
/__w/aspect/aspect/source/main.cc:517:49: error: ignoring return value of 'T dealii::Utilities::MPI::broadcast(MPI_Comm, const T&, unsigned int) [with T = bool; MPI_Comm = ompi_communicator_t*]', declared with attribute nodiscard [-Werror=unused-result]
  517 |               dealii::Utilities::MPI::broadcast (MPI_COMM_WORLD,
      |               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~
  518 |                                                  /* reading failed: */ false,
      |                                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  519 |                                                  root_process);
      |                                                  ~~~~~~~~~~~~~
In file included from /__w/aspect/aspect/include/aspect/global.h:28,
                 from /__w/aspect/aspect/build/CMakeFiles/aspect.exe.debug.dir/cmake_pch.hxx:5,
                 from <command-line>:
/opt/deal.II-master/include/deal.II/base/mpi.h:2177:5: note: declared here
 2177 |     broadcast(const MPI_Comm     comm,
      |     ^~~~~~~~~
In file included from /__w/aspect/aspect/build/CMakeFiles/aspect.exe.debug.dir/Unity/unity_0_cxx.cxx:3:
/__w/aspect/aspect/source/main.cc:522:45: error: ignoring return value of 'T dealii::Utilities::MPI::broadcast(MPI_Comm, const T&, unsigned int) [with T = bool; MPI_Comm = ompi_communicator_t*]', declared with attribute nodiscard [-Werror=unused-result]
  522 |           dealii::Utilities::MPI::broadcast (MPI_COMM_WORLD, true, root_process);
      |           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /__w/aspect/aspect/include/aspect/global.h:28,
                 from /__w/aspect/aspect/build/CMakeFiles/aspect.exe.debug.dir/cmake_pch.hxx:5,
                 from <command-line>:
/opt/deal.II-master/include/deal.II/base/mpi.h:2177:5: note: declared here
 2177 |     broadcast(const MPI_Comm     comm,
      |     ^~~~~~~~~
```
This seems to be new. How did that not trigger before? This is code that was merged in mid-April...

--- a/source/main.cc
+++ b/source/main.cc
@@ -30,6 +30,7 @@
 #include <string>
 #include <thread>
 #include <regex>
+#include <utility>
 
 #ifdef DEBUG
 #ifdef ASPECT_USE_FP_EXCEPTIONS
@@ -514,12 +515,12 @@ parse_parameters (const std::string &input_as_string,
             }
           catch (const dealii::ExceptionBase &e)
             {
-              dealii::Utilities::MPI::broadcast (MPI_COMM_WORLD,
-                                                 /* reading failed: */ false,
-                                                 root_process);
+              std::ignore = dealii::Utilities::MPI::broadcast (MPI_COMM_WORLD,
+                                                               /* reading failed: */ false,
+                                                               root_process);
               throw;
             }
-          dealii::Utilities::MPI::broadcast (MPI_COMM_WORLD, true, root_process);
+          std::ignore = dealii::Utilities::MPI::broadcast (MPI_COMM_WORLD, true, root_process);
         }
       else
         {

